### PR TITLE
Clean up duplicate constants

### PR DESCRIPTION
**Pull request by Boreeas**

### CODE REVIEW

File: src/main/java/cubicchunks/util/CubeCoords.java [hunk 1]
```diff
@@ -31,16 +31,11 @@
...
-import static cubicchunks.util.Coords.CUBE_MAX_Y;
-import static cubicchunks.util.Coords.CUBE_MAX_Z;
-import static cubicchunks.util.Coords.HALF_CUBE_MAX_X;
-import static cubicchunks.util.Coords.HALF_CUBE_MAX_Y;
-import static cubicchunks.util.Coords.HALF_CUBE_MAX_Z;
 import static cubicchunks.util.Coords.blockToCube;
 import static cubicchunks.util.Coords.getCubeXForEntity;
 import static cubicchunks.util.Coords.getCubeYForEntity;
 import static cubicchunks.util.Coords.getCubeZForEntity;
+import static cubicchunks.world.cube.Cube.SIZE;
```
Comment (Cyclonit) [inline src/main/java/cubicchunks/util/CubeCoords.java]: SIZE itself does not have any fixed meaning with respect to CubeCoords and Cube.SIZE isn't much longer.

Comment (Boreeas) [inline src/main/java/cubicchunks/util/CubeCoords.java]: You're right, fixed

File: src/main/java/cubicchunks/worldgen/generator/custom/CustomTerrainGenerator.java [hunk 2]
```diff
@@ -34,9 +34,7 @@
 
 import java.util.Random;
 
-import static cubicchunks.util.Coords.CUBE_MAX_X;
-import static cubicchunks.util.Coords.CUBE_MAX_Y;
-import static cubicchunks.util.Coords.CUBE_MAX_Z;
+import static cubicchunks.world.cube.Cube.SIZE;
```
Comment (Cyclonit) [inline src/main/java/cubicchunks/worldgen/generator/custom/CustomTerrainGenerator.java]: SIZE doesn't have any proper meaning by itself here either.

File: src/main/java/cubicchunks/worldgen/generator/custom/structures/CubicCaveGenerator.java [hunk 3]
```diff
@@ -35,7 +35,7 @@
 import java.util.Random;
 import java.util.function.Predicate;
 
-import static cubicchunks.util.Coords.CUBE_SIZE;
+import static cubicchunks.world.cube.Cube.SIZE;
```
Comment (Cyclonit) [inline src/main/java/cubicchunks/worldgen/generator/custom/structures/CubicCaveGenerator.java]: Cube.SIZE is more meaningful and actually being used throughout this class.

File: src/main/java/cubicchunks/world/cube/Cube.java [hunk 4]
```diff
@@ -56,14 +56,13 @@
...
 import org.apache.logging.log4j.Logger;
 
+import javax.annotation.Nullable;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ConcurrentLinkedQueue;
 
-import javax.annotation.Nullable;
```
Comment (Boreeas) [inline src/main/java/cubicchunks/world/cube/Cube.java]: I guess IDEA decided to remove unneeded imports when I renamed the variables?

Comment (Barteks2x) [inline src/main/java/cubicchunks/world/cube/Cube.java]: I meant that the import it moved up, but github didn't show it here

Comment (Boreeas) [inline src/main/java/cubicchunks/world/cube/Cube.java]: Same reason, I figure. I refactored stuff, and IDEA uses that opportunity to reformat imports. Should be according to the project's format settings either way.

**Pull request merged**